resources/migrations/000_migrations.yaml

```diff
@@ -7376,7 +7361,7 @@ databaseChangeLog:
             referencedTableName: collection
             referencedColumnNames: id
             constraintName: fk_snippet_collection_id
-            onDelete: SET NULL
+            onDelete: CASCADE
```

Comment:
@camsaul double-checking that this is appropriate here. It seems like if the original addColumn had `onDelete: CASCADE` then we would want that in the ad-hoc FK change here, right?

Reply:
This is a good catch and a good question. I think looking at all of our other `collection_id` foreign keys the general behavior we want is actually `SET NULL` -- if the Collection something is in gets deleted, move things in that Collection to the "Root Collection" (`collection_id = nil`) as opposed to deleting them altogether.

So I think the latter migration is correct and the original one was the one that was wrong.

Reply:
That's just my theory. Either way, I don't want to second-guess whatever wacky reasoning I had at the time and forgot to write down -- it's definitely safer not to change this behavior than it is to change it. So I'd vote just to leave things as-is for now